run_operations.py

```diff
@@ -1,394 +1,293 @@
-#!/usr/bin/env python3
-
+#!/usr/bin/env python3 
 import os
 import sys
 import subprocess
 import argparse
-from datetime import datetime
+from datetime import date, datetime
 import shutil
 import time
 import glob
 
-import subprocess
-################### directory initiation for Josh to review (and remove comment)
-##################
-
 import generate_template_files as gt
+from rsmas_logging import RsmasLogger, loglevel
+from dataset_template import Template
+import _process_utilities as putils
 
-import logging
+OPERATIONS_DIRECTORY = os.getenv('OPERATIONS')
+STORED_DATE_FILE = OPERATIONS_DIRECTORY + "/stored_date.date"
+TEMPLATE_DIRECTORY = OPERATIONS_DIRECTORY + "/TEMPLATES"
+DATE_FORMAT = "%Y-%m-%dT%H:%M:%S.%f"
 
-#################### LOGGERS AND LOGGING SETUP ####################
+logger_file = "{}/{}/run_operations.log".format(OPERATIONS_DIRECTORY, "LOGS")
+logger_run_operations = RsmasLogger(logger_file)
 
-logger = logging.getLogger()
-logger.setLevel(logging.DEBUG)
+def create_process_rsmas_parser():
```

Comment:
Move parser to end of file. Move most relevant code to top of file for easing understanding.

Reply:
@2gotgrossman Im going to leave this for a different diff as it should be done across the entire codebase at once